docs/usage/configuration-options.md

```diff
@@ -2134,9 +2134,29 @@ Managers which do not support replacement:
 - `regex`
 
 Use the `replacementName` config option to set the name of a replacement package.
-Must be used with `replacementVersion` (see example below).
+
+Can be used in combination with `replacementVersion`, `replacementPrefixAdd` and `replacmeentPrefixRemove`.
```

Comment:
```suggestion
Can be used in combination with `replacementVersion`, `replacementPrefixAdd` and `replacementPrefixRemove`.
```
Seems to be a copy and paste error in the docs

Reply:
Good eyes!